test/e2e/app-dir/error-on-next-codemod-comment/error-on-next-codemod-comment.test.ts

```diff
@@ -0,0 +1,125 @@
+import { nextTestSetup } from 'e2e-utils'
+import {
+  assertHasRedbox,
+  assertNoRedbox,
+  getRedboxSource,
+  retry,
+} from 'next-test-utils'
+
+describe('app-dir - error-on-next-codemod-comment', () => {
+  const { next, isNextDev } = nextTestSetup({
+    files: __dirname,
+    skipStart: true,
+    skipDeployment: true,
+  })
+
+  if (isNextDev) {
+    beforeAll(async () => {
+      await next.start()
+    })
+
+    it('should error with swc if you have codemod comments left', async () => {
+      const browser = await next.browser('/')
+
+      await assertHasRedbox(browser)
+
+      if (process.env.TURBOPACK) {
+        expect(await getRedboxSource(browser)).toMatchInlineSnapshot(`
+          "./app/page.tsx:2:2
+          Ecmascript file had an error
+            1 | export default function Page() {
+          > 2 |   // @next-codemod-error remove jsx of next line
+              |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
aside: column is off by one here. We also have other tests where we observe this. Just making a mental note for when we revisit the error overlay.